### DISCUSSION TITLE
keep getting read only file when installing

### DISCUSSION THREAD

**Discussion by jonahgcarpenter** [exadel-inc/CompreFace#657] [Q&A]
Im trying to install compreface on docker desktop inside a windows 10 vm on my unraid server. Everytime i try to install docker gives me an error " failed to register layer: Error processing tar file(exit status 1): write /app/m1/.modules/agegender/gender/21936/checkpoint-14999.data-00000-of-00001: read-only file system " any help?

**Comment by pospielov** [comment]
I looked at the unraid forums and just googled "read-only file system docker windows" and it looks like there are several problems that could lead to this error:
1. Corrupt image or something like that, they recommend just recreating it
2. Lack of disk space - just check it
3. corrupted file system
Not sure what the problem is in your case. Does it reproduce only on CompreFace docker images?
